vignettes/SEIRDV.Rmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "The SEIRDV model"
+author: "Ioana Bouros"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{The SEIRDV model}
+  %\VignetteEngine{knitr::knitr}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(comomodels)
+library(tidyverse)
+library(ggplot2)
+library(glue)
+library(dplyr)
+```
+
+## Introduction
+This document explains the basics of the SEIRV model. The model describes
+how populations of susceptible, exposed, infectious, recovered and vaccinated
+individuals evolve over time. Waning immunity is also considered in this model,
+so there is a constant supply of susceptibles in the population which makes the model
+more realistic.
+
+## The SEIRDV Model
+The SEIRDV model consists of five ODEs describing five populations of susceptible,
+recovered, infectious, recovered and vaccinated individuals. Susceptible
+individuals ($S$) have not been exposed to the virus. Exposed individuals ($E$) have been
+exposed to the virus and have been infected but are not yet infectious to others; they are 
+in the incubation period. Infectious individuals ($I$) can spread the virus to 
+others.  Recovered individuals ($R$) are no longer
+infectious, but they can return to the susceptible population when they lose their
+immunity. Vaccinated individuals ($V$) cannot become infected and are not ill at
+the time, but as it was the case with the recovered population, their immunity
+is temporary and they can become susceptible again with a certain rate. 
+We can vaccinate not only the susceptibles, but also $E$, $I$ and $R$s. However,
+they have already been exposed to the illness so they do not change compartment
+assignment. Births and deaths unrelated to the infection are not considered in
+this model. The ODEs describing how the five populations evolve over time are
+$$\frac{\text{d}S}{\text{d}t} = -\beta S I - \nu S + \delta_V V + \delta_R R,$$
```

Comment:
Not sure if this is something that was considered---is there any point to nu anymore? why not just have the rate of vaccination be one of the intervention parameters directly instead of the 0--1 coverage?
If you keep nu it should be redefined as a maximum possible vaccination rate.

Reply:
I will do exactly that! Thanks for the suggestion